examples/flutter_gallery/lib/demo/cupertino/cupertino_buttons_demo.dart

```diff
@@ -19,64 +18,68 @@ class _CupertinoButtonDemoState extends State<CupertinoButtonsDemo> {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: const Text('Cupertino Buttons'),
-        actions: <Widget>[MaterialDemoDocumentationButton(CupertinoButtonsDemo.routeName)],
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        middle: const Text('Buttons'),
+        previousPageTitle: 'Cupertino',
```

Comment:
Is there a reason that we have to explicitly define the previous page title in all the demos?

Reply:
Good point, I should point it out. Added comments.